src/ucp/wireup/wireup_ep.h

```diff
@@ -39,6 +39,8 @@ struct ucp_wireup_ep {
     volatile uint32_t         pending_count; /**< Number of pending wireup operations */
     volatile uint32_t         flags;         /**< Connection state flags */
     uct_worker_cb_id_t        progress_id;   /**< ID of progress function */
+    ucp_ep_params_t           ucp_ep_params; /**< UCP EP paramters */
+    char                      dev_name[UCT_DEVICE_NAME_MAX];
```

Comment:
why do we need to save dev_name? can just pass it ucp_cm_preinit_client_lanes

Reply:
fixed in #4283